web/bot.py

```diff
@@ -0,0 +1,148 @@
+import logging
```

Comment:
could you please rename filename to telegram_bot.py?

Reply:
Done